### DISCUSSION TITLE
How to remove top padding?

### DISCUSSION THREAD

**Discussion by oclcgg** [patrykandpatrick/vico#698] [Questions] [answered]
I tried adding background color and surface parent to CartesianChartHost (version vico-2.0.0-alpha.19). However, seems like padding is added by default to the chart as shown in the red box below:

![vico_padding](https://github.com/patrykandpatrick/vico/assets/169393973/b7def484-f184-466e-bed7-9cf23c99f784)

Is there a way to remove the top padding?

**Comment by Gowsky** [comment, marked as answer]
Hello, please see [here](https://github.com/patrykandpatrick/vico/discussions/567#discussioncomment-8455664). `Marker` is now `CartesianMarker` and `MarkerComponent` is now `DefaultCartesianMarker`.
